src/styles/options.scss

```diff
@@ -4,18 +4,101 @@
     margin: 10px;
 }
 
+.toggle-switch {
```

Comment:
Good candidate to spin out into a custom element at some point. Seems a bit overkill, but could be fun.